docs/source/installation.rst

```diff
@@ -64,32 +63,33 @@ Release versions - troubleshooting
 Module not found
 """"""""""""""""
 
-The most frequent reason for "module not found" errors is installing ``sktime`` with minimum dependencies
-and using an estimator which interfaces a packages that has not been installed in the environment.
+The most frequent reason for *module not found* errors is installing ``sktime`` with minimum dependencies
+and using an estimator which interfaces a package that has not been installed in the environment.
 To resolve this, install the missing package, or install ``sktime`` with maximum dependencies (see above).
 
 
 Facebook prophet
 """"""""""""""""
 
 A frequent issue arises with installation of facebook prophet when installing via ``pip``, especially on Windows systems.
+There are several workarounds:
 
-Potential workaround no.1, install manually via ``conda-forge``:
+1. Install manually via ``conda-forge``:
```

Comment:
changing it to the list may confuse users and suggest they need to execute these sequentially, as opposed to alternatively. I suggest bringing back the original formulation.

Reply:
this is because "1., 2., 3." is strongly interpreted as a sequence.